tests/RobotFramework/libraries/ThinEdgeIO/ThinEdgeIO.py

```diff
@@ -220,9 +220,9 @@ def get_logs(
 
         try:
             # tedge-mapper-c8y message log (if they exist)
-            log.info("tedge-mapper-c8y message log: /etc/tedge/.tedge-mapper-c8y/entity_store.jsonl")
+            log.info("tedge-mapper-c8y message log: /etc/tedge/.agent/entity_store.jsonl")
             device.execute_command(
-                "cat /etc/tedge/.tedge-mapper-c8y/entity_store.jsonl || true",
+                "cat /etc/tedge/.agent/entity_store.jsonl || true",
```

Comment:
It might be worthwhile to collect both files (in case if we're running the tests on a remove instance). With a minimal change it will work for both the new and old (without breaking the compatibility with anything).

You can use `head -n-1` will actually printed out a nice result which also includes the name of the file (if it exists).

```sh
head -n-1 /etc/tedge/.tedge-mapper-c8y/entity_store.jsonl /etc/tedge/.agent/entity_store.jsonl 2>/dev/null || true
```

Reply:
Below shows an example of the output (though using different files...but it shows what the header looks like):

```sh
$ head -n-1 /etc/tedge/tedge.toml /etc/tedge/system.toml 2>/dev/null || true
==> /etc/tedge/tedge.toml <==
[config]
version = "2"


==> /etc/tedge/system.toml <==
[init]
name = "tedgectl"
is_available = ["/usr/bin/tedgectl", "is_available"]
restart = ["/usr/bin/tedgectl", "restart", "{}"]
stop =  ["/usr/bin/tedgectl", "stop", "{}"]
enable =  ["/usr/bin/tedgectl", "enable", "{}"]
disable =  ["/usr/bin/tedgectl", "disable", "{}"]
is_active = ["/usr/bin/tedgectl", "is_active", "{}"]

[system]
```